src/composables/transactionData.ts

```diff
@@ -109,6 +116,25 @@ export function useTransactionData({
     ),
   );
 
+  const isTokenSwap = computed(
+    (): boolean => tokenContractAddresses.value
+      .concat(tokenSwapAddresses.value).some((address) => address === innerTx.value?.contractId),
```

Comment:
I think this is more readable:
```suggestion
    (): boolean => [...tokenContractAddresses.value, ...tokenSwapAddresses.value].some((address) => address === innerTx.value?.contractId),
```